overlord/snapstate/snapstate.go

```diff
@@ -64,11 +68,21 @@ func (f Flags) JailMode() bool {
 	return f&JailMode != 0
 }
 
+func (f Flags) HasABA() bool {
+	return f&HasABA != 0
+}
+
+func (f Flags) IsRevert() bool {
```

Comment:
also we wouldn't need this helper if we made the constant SnapSetupFlags only (if we can)